### PR TITLE
feat(vdp): Update endpoints.json with new streaming endpoints

### PR DESCRIPTION
**Pull request by tillknuesting**
Because

- new endpoints for streaming are introduced for vdp

This commit

- adds the endpoints to the config

### CODE REVIEW

File: config/settings-env/endpoints.json [hunk 1]
```diff
@@ -550,6 +550,20 @@
...
       },
+      {
+        "endpoint": "/v1beta/users/{user_id}/pipelines/{pipeline_id}/trigger:stream",
+        "url_pattern": "/v1beta/users/{user_id}/pipelines/{pipeline_id}/trigger:stream",
+        "method": "POST",
+        "timeout": "600s",
+        "input_query_strings": []
+      },
+      {
+        "endpoint": "/v1beta/sse",
```
Comment (tillknuesting) [inline config/settings-env/endpoints.json:561]: it is unclear to what extend krakend actually supports SSE, probably have to develop a plugin for it

File: config/settings-env/endpoints.json [hunk 2]
```diff
@@ -550,6 +550,20 @@
...
+        "url_pattern": "/v1beta/sse",
```
Comment (jvallesm) [inline config/settings-env/endpoints.json:562]: Will this endpoint be called after the trigger one? If so, will the trigger ID be passed as a header, or how would it work? If you could outline the sequence of requests and/or attach a small diagram it'd be great.

**Comment by tillknuesting** [comment]
After testing it, I found that out of the box, KrakenD waits until the buffer is full or the connection is closed, hence streaming that would be normally flushing after each message is defeated.

File: config/settings-env/endpoints.json [hunk 3]
```diff
@@ -550,6 +550,20 @@
         "timeout": "600s",
         "input_query_strings": []
       },
+      {
+        "endpoint": "/v1beta/users/{user_id}/pipelines/{pipeline_id}/trigger:stream",
```
Comment (pinglin) [inline config/settings-env/endpoints.json:554]: Does this endpoint actually work? The thins is KrakenD doesn't support colon in the endpoint URL ([ref](https://gophers.slack.com/archives/CANCJD0V8/p1656523770018559?thread_ts=1656462092.785329&cid=CANCJD0V8)).

**Comment by donch1989** [comment]
> After testing it, I found that out of the box, KrakenD waits until the buffer is full or the connection is closed, hence streaming that would be normally flushing after each message is defeated.

Do you mean that on the client side, the client will receive all messages at once instead of one by one?

**Comment by tillknuesting** [comment]
> > After testing it, I found that out of the box, KrakenD waits until the buffer is full or the connection is closed, hence streaming that would be normally flushing after each message is defeated.
> 
> Do you mean that on the client side, the client will receive all messages at once instead of one by one?

Yes, the http client calling via the API gateway will receive the messages all together according to the flush interval and disconnect after the response.

**Comment by tillknuesting** [comment]
will open seperate PR with new plugin and adjusted configs

**Pull request closed without merging**